src/clients/requestFilter.ts

```diff
@@ -0,0 +1,22 @@
+import Redis from 'ioredis';
+
+const redis = new Redis(process.env.REDIS_URL);
+const anHourInMilliSecond = 60 * 60 * 1000;
+
+// Check whether the requester can request Faucet or not based on the last request time.
+export const canRequestFaucet = async (requesterId: string, now: number): Promise<void> => {
+    const lastRequestAt = Number(await redis.get(requesterId));
+    const untilNextRequest = lastRequestAt + anHourInMilliSecond - now;
```

Comment:
I cannot see how long the `untilNextRequest` is supposed to be. I suggest creating a constant that defines how long the cooldown is.
For example:
```ts
// must wait 30 minutes before the next request
const cooldown = 30 * 60 * 1000
```